pandas-stubs/core/indexes/interval.pyi

```diff
@@ -47,37 +69,57 @@ class IntervalIndex(IntervalMixin, ExtensionIndex):
     @classmethod
     def from_tuples(
         cls,
-        data,
+        data: Sequence[tuple[int, int]]
+        | Sequence[tuple[float, float]]
+        | Sequence[tuple[DatetimeLike, DatetimeLike]]
+        | npt.NDArray,
         closed: IntervalClosedType = ...,
         name: Hashable = ...,
         copy: bool = ...,
         dtype: IntervalDtype | None = ...,
     ) -> IntervalIndex: ...
+    def __contains__(self, key: Any) -> bool: ...
     def astype(self, dtype: DtypeArg, copy: bool = ...) -> IntervalIndex: ...
     @property
     def inferred_type(self) -> str: ...
     def memory_usage(self, deep: bool = ...) -> int: ...
+    def is_monotonic_decreasing(self) -> bool: ...
+    def is_unique(self) -> bool: ...
     @property
     def is_overlapping(self) -> bool: ...
-    def get_loc(self, key, tolerance=...) -> int | slice | np.ndarray: ...
+    # Note: tolerance removed as it has no effect
+    def get_loc(
+        self,
+        key: Label,
+        method: str | None = ...,
```

Comment:
`FillnaOptions | Literal['nearest'] | None`

Could fix it in `core/indexes/base.pyi` as well.

Reply:
Fixed in both.